script.js

```diff
@@ -1,121 +1,145 @@
-var input = document.querySelectorAll('input');
+var myLibrary = [];
 
-let newBook = document.querySelector('#add-new-book');
-let modal = document.querySelector('#modal');
+let library = document.getElementById('container');
+let addBookButton = document.getElementById('add-new-book');
+let form = document.getElementById('new-book');
+let modal = document.getElementById('modal');
 let overlay = document.querySelector('.overlay');
-let submitBook = document.querySelector('#submit-book');
-let container = document.querySelector('.container');
-let form = document.querySelector('form');
 
+addBookButton.addEventListener('click', toggleModal);
+overlay.addEventListener('click', toggleModal);
 
-newBook.addEventListener('click', addNewBook);
-overlay.addEventListener('click', closeModal);
-submitBook.addEventListener('click', bookDetails);
+const addBookToLibrary = (e) => {
+  e.preventDefault();
+  let bookData = [];
```

Comment:
This `bookData` has no shape which could lead to data mismatches if something in the app changes. This should be an object with defined props that are modified with their matching input values.